test/openshift-test.js

```diff
@@ -0,0 +1,68 @@
+const test = require('tape');
+const OpenshiftTestAssistant = require('openshift-test-assistant');
+const openshiftAssistant = new OpenshiftTestAssistant();
+const path = require('path');
+const request = require('supertest');
+
+test('setup', (t) => {
```

Comment:
As a general rule, I like to use `t.plan(n)` to specify the test expectations. With asynchronous code, this can sometimes catch tests that aren't being run for one reason or another. l'd recommend adding that to each of your tests (and setup/teardown).